packages/app/src/app/pages/Sandbox/Editor/Content/index.js

```diff
@@ -424,7 +424,9 @@ class EditorPreview extends React.Component {
     const browserConfig = {
       id: 'codesandbox.browser',
       title: options =>
-        options.port ? `Browser (:${options.port})` : `Browser`,
+        options.port
```

Comment:
Need to have a check here for title, since now it doesn't work if only title is given.

Reply:
pushed